_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -141,27 +184,30 @@ Heading marked up with `role="heading"` that does not describe the topic or purp
 
 #### Failed example 4
 
-Heading marked up with `h` element that does not describe the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
+The heading marked up with `h` element does not describe the topic or purpose of its section of the content. The heading is visible, but is not included in the Accessibility Tree.
 
 ```html
-<h1 class="target" aria-hidden="true">Weather</h1>
+<h1 aria-hidden="true">Weather</h1>
 <p>We are open Monday through Friday from 10 to 16</p>
 ```
 
 #### Failed example 5
 
-Empty heading marked up with `h` element.
+The heading marked up with `h` element has visible text nodes that describe the content, but the accessible name doesn't.
```

Comment:
Grammar ... should this read "visible text nodes" or "a visible text node"?

Reply:
There can apparently be more text nodes, so we are going with the plural form.